kornia/utils/_compat.py

```diff
@@ -33,12 +33,9 @@ def torch_version_ge(major: int, minor: int, patch: int) -> bool:
 
 
 if version.parse(torch_version()) > version.parse("1.7.1"):
-    # TODO: remove the type: ignore once Python 3.6 is deprecated.
-    # It turns out that Pytorch has no attribute `torch.linalg` for
-    # Python 3.6 / PyTorch 1.7.0, 1.7.1
-    from torch.linalg import qr as linalg_qr  # type: ignore
+    from torch.linalg import qr as linalg_qr
```

Comment:
actually we might want to remove the if else version ?

Reply:
Yes, this `_compat.py` need to be updated/reviewed, but I don't know what should be kept -- some errors of typing is from this file too

Reply:
i guess because of old interfaces of qr, solve, etc that pytorch changed over time. I believe that now are more mature and worth to upfate